src/dipdup/config.py

```diff
@@ -908,6 +920,24 @@ def address_filter(self) -> Set[str]:
         return addresses
 
 
+@dataclass
+class OperationUnfilteredIndexConfig(IndexConfig):
+    """Operation index config
+
+    :param kind: always `operation`
+    :param handlers: List of indexer handlers
+    :param first_level: Level to start indexing from
+    :param last_level: Level to stop indexing at (DipDup will terminate at this level)
+    """
+
+    kind: Literal["operation_unfiltered"]
+    handlers: Tuple[OperationUnfilteredHandlerConfig, ...]
+    types: Tuple[OperationType, ...] = (OperationType.transaction,)
```

Comment:
Should prob be `OperationType.origination` since we support originations atm only